client/src/components/browseUserRecipes/Category.svelte

```diff
@@ -0,0 +1,95 @@
+<script>
+  import { createEventDispatcher, onMount } from 'svelte';
+  const dispatch = createEventDispatcher();
+
+  function handleChange({ target }) {
+    dispatch('categoryToggle', {
+      value: target.value,
+      text: target.options[target.selectedIndex].text,
+    });
+  }
```

Comment:
this is the same component as Category, just make it a common component please. When we talked I thought you meant just some part of the logic, not this as well 😞